<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>746</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>702</width>
    <height>746</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>702</width>
    <height>746</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #2c3e50; /* Azul oscuro elegante */
}

QPushButton {
    background-color: #3498db; /* Azul claro */
    color: white;
    font-size: 14px;
    border-radius: 5px;
    padding: 6px 12px;
}

QPushButton:hover {
    background-color: #2980b9;
}

QPushButton:pressed {
    background-color: #1f6690;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>311</width>
      <height>301</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>291</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="informe">
       <property name="text">
        <string>Informe 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="informe2">
       <property name="text">
        <string>Informe 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="informe3">
       <property name="text">
        <string>Informe 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="informe4">
       <property name="text">
        <string>Informe 4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="informe5">
       <property name="text">
        <string>Informe  5</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>label</zorder>
   <zorder>informe</zorder>
   <zorder>informe2</zorder>
   <zorder>informe3</zorder>
   <zorder>informe4</zorder>
   <zorder>informe5</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
